Client/src/modules/admin/CacheSettings.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <q-page class="page-padding">
+    <div class="header-with-button">
+      <h2 class="q-title">{{$tl("title")}}</h2>
+      <div class="clear"></div>
+    </div>
+    <div>
+      <div class="q-mt-lg">
+        <q-select
+          v-model="policy"
+          :label="$tl('CachePolicy')"
+          :options="PolicyTypes"
+        />
+        <br />
+        <q-input v-if="policy !== null && policy.id !== 1" v-model="cacheSettings.invalidateCacheTime" type="number" :label="$tl('CacheLifetime')" />
+      </div>
+
+      <br>
+      <div class="btn-block">
+        <q-btn icon="fas fa-plus" class="btn-send" :loading="loading"
+          no-caps :label="$tl('SaveChanges')" color="send"
+          @click="save">
+          <LoaderSent slot="loading"/>
+        </q-btn>
+      </div>
+    </div>
+  </q-page>
+</template>
+
+<script>
+import Page from "Page";
```

Comment:
Is it okay have scripts alongside html tags? I believe it's considered better to have separate files for such scripts, correct me if I'm wrong.

Reply:
It depends on situations.
If big algorithms or shared code its good to make separate.
In other cases my opinion is that best way to make inside Vue component file.